lib/foreman_maintain/concerns/downstream.rb

```diff
@@ -101,22 +122,42 @@ def puppet4_repo(full_version)
       end
 
       def common_repos(full_version)
-        repos_arrary = if ENV['FOREMAN_MAINTAIN_USE_BETA'] == '1'
-                         ["rhel-#{el_major_version}-server-satellite-maintenance-6-beta-rpms",
-                          "rhel-#{el_major_version}-server-satellite-tools-6-beta-rpms"]
-                       else
-                         ["rhel-#{el_major_version}-server-satellite-maintenance-6-rpms",
-                          "rhel-#{el_major_version}-server-satellite-tools-#{full_version}-rpms"]
-                       end
+        sat_maint_version = '6'
```

Comment:
The sat_maint_version should be 6 upto Satellite or Capsule version 6.10. Since 7.0 it should be `full_version`.